### PR TITLE
bump versions for sdk/sync sdk/iptb and ipfs/testground

### PR DESCRIPTION
**Pull request by nonsense**

### CODE REVIEW

File: sdk/sync/go.mod [hunk 1]
```diff
@@ -4,13 +4,17 @@ go 1.14
 
 require (
 	github.com/go-redis/redis/v7 v7.2.0
-	github.com/ipfs/testground v0.1.0
-	github.com/ipfs/testground/sdk/runtime v0.3.0
-	github.com/libp2p/go-libp2p-core v0.2.3
-	github.com/libp2p/go-openssl v0.0.4 // indirect
-	github.com/multiformats/go-multiaddr v0.1.1
+	github.com/ipfs/testground v0.4.0
```
Comment (raulk) [inline sdk/sync/go.mod:7]: Mind running a gomod tidy here?

Comment (nonsense) [reply sdk/sync/go.mod:7]: this is after `go mod tidy`.

Comment (nonsense) [reply sdk/sync/go.mod:7]: I believe we have it in CI, so if `go mod tidy` produces something else, CI fails.

File: plans/placebo/go.mod [hunk 2]
```diff
@@ -2,6 +2,10 @@ module github.com/ipfs/testground/plans/placebo
 
 go 1.14
 
-require github.com/ipfs/testground/sdk/runtime v0.3.0
+require (
+	github.com/ipfs/testground/sdk/runtime v0.4.0
+	github.com/pkg/errors v0.9.1 // indirect
+	golang.org/x/tools v0.0.0-20200227222343-706bc42d1f0d // indirect
+)
```
Comment (raulk) [inline plans/placebo/go.mod:9]: go mod seems broken; I don't know how this happened. If you remove the indirect dependencies and run go mod tidy, it doesn't add them 🤷‍♂️

Comment (nonsense) [reply plans/placebo/go.mod]: It was happening on the `plans` for me as well. I am on Go 1.13.1, that might be the case, will upgrade now.

**Comment by raulk** [comment]
go mod is doing weird things, leaking indirect dependencies all over the place. If you remove them manually and run go mod tidy again, they don't reappear 🤷‍♂️

**Pull request merged**